DataFormats/SiPixelDetId/interface/PixelChannelIdentifier.h

```diff
@@ -2,6 +2,7 @@
 #define DATAFORMATS_PIXELCHANMNELIDENTIFIER_H
```

Comment:
```suggestion
#define  DataFormats_SiPixelDetId_interface_PixelChannelIdentifier_h
```

...or at least fix the typo in `CHANMEL` :-)